### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by l98y3j26** [deepmodeling/dpgen#507] [Q&A]
Dear developers,

When I run the DPGEN on the Lebesgue, here comes the errors and the param.json and machine.json are attached.
[param.txt](https://github.com/deepmodeling/deepmd-kit/files/6965510/param.txt)
[machine.txt](https://github.com/deepmodeling/deepmd-kit/files/6965511/machine.txt)
 

![8a65437a8cabdcd4a2d479dc1d29b29](https://user-images.githubusercontent.com/86098920/129181441-ba6f8436-e20c-444c-9cf4-0381ad3bdf7a.jpg)

**Comment by AnguseZhang** [comment]
Can you make sure that your initial data `data.init/deepmd` is a proper DeePMD-kit's format?
See explanations in https://github.com/deepmodeling/deepmd-kit/blob/master/doc/use-deepmd-kit.md#prepare-data

You can use `dpdata.LabeledSystem("data.init/deepmd")` to try to load data
